rcl/test/rcl/test_wait.cpp

```diff
@@ -205,8 +210,14 @@ TEST_F(CLASSNAME(WaitSetTestFixture, RMW_IMPLEMENTATION), canceled_timer) {
   ret = rcl_wait_set_add_guard_condition(&wait_set, &guard_cond);
   EXPECT_EQ(RCL_RET_OK, ret) << rcl_get_error_string_safe();
 
+  rcl_clock_t clock;
+  rcl_allocator_t allocator = rcl_get_default_allocator();
+  ret = rcl_clock_init(RCL_STEADY_TIME, &clock, &allocator);
```

Comment:
Missing `rcl_clock_fini()`, though I wouldn't block the PR for that in a test.

Reply:
Added in 1c73377792db3b854a83584557cd92c53e9cdd07.